framework/math/quadratures/angular/sldfe_sq_quadrature.cc

```diff
@@ -787,181 +743,84 @@ SimplifiedLDFESQ::Quadrature::TestIntegration(int test_case, double ref_solution
   char buff0[200], buff1[200], buff2[200];
   snprintf(buff0, 200, "Riemann integral: %.20e\n", I_riemann);
   snprintf(buff1, 200, "Quadrature integral: %.10e\n", I_quadrature);
-  snprintf(buff2,
-           200,
-           "Error_RQ%05d_%06d: %2d %f %e\n",
-           Nd,
-           Nd * 8,
-           initial_level_,
-           h,
+  snprintf(buff2, 200, "Error_RQ%05d_%06d: %2d %f %e\n", Nd, Nd * 8,
+           initial_level_, h,
            std::fabs((I_riemann - I_quadrature) / ref_solution));
 
   log.Log() << buff0;
   log.Log() << buff1;
   log.Log() << buff2;
 }
 
-void
-SimplifiedLDFESQ::Quadrature::PrintQuadratureToFile(const std::string& file_base)
-{
+void SimplifiedLDFESQ::Quadrature::PrintQuadratureToFile(
+    const std::string &file_base) {
   log.Log() << "Printing SLDFE-Quadrature to file.";
 
-  std::ofstream vert_file, cell_file, points_file, python_file;
-  vert_file.open(file_base + "verts.txt");
+  std::ofstream vert_file, cell_file, points_file;
+
+  // Vertex file generation for plotting - is written to the path above
+  vert_file.open(file_base + "verts.csv");
   {
-    for (const auto& sq : deployed_SQs)
-      for (int v = 0; v < 4; ++v)
-      {
-        auto& v0 = sq.vertices_xyz_prime[v];
-        auto& v1 = sq.vertices_xyz_prime[(v < 3) ? v + 1 : 0];
+    vert_file << "x,y,z\n"; // CSV Header
+    for (const auto &sq : deployed_SQs) {
+      for (int v = 0; v < 4; ++v) {
+        auto &v0 = sq.vertices_xyz_prime[v];
+        auto &v1 = sq.vertices_xyz_prime[(v < 3) ? v + 1 : 0];
 
-        for (int d = 0; d <= 10; ++d)
-        {
+        for (int d = 0; d <= 10; ++d) {
           auto vert = (1.0 - d / 10.0) * v0 + (d / 10.0) * v1;
           vert = vert * sq.octant_modifier;
           vert.Normalize();
-          vert_file << vert.x << " " << vert.y << " " << vert.z << "\n";
+          vert_file << vert.x << "," << vert.y << "," << vert.z << "\n";
         }
       }
+    }
   }
   vert_file.close();
 
-  cell_file.open(file_base + "cells.txt");
+  // Indexing file for polygons for plotting
+  cell_file.open(file_base + "cells.csv");
```

Comment:
i would add `_` in front of points, so that the users do not need to provide a basename that includes `_`